src/js/components/dashboard/impacts/ImpactGauge.jsx

```diff
@@ -0,0 +1,58 @@
+/**
+ * ImpactGauge.jsx
+ * Created by Daniel Boos 4/1/2020
+ */
+
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    submissionData: PropTypes.object,
+    openModal: PropTypes.func.isRequired,
+    level: PropTypes.string
+};
+
+const defaultProps = {
+    submissionData: {},
+    level: 'high'
+};
+
+export default class ImpactGauge extends React.Component {
+    constructor(props) {
+        super(props);
+
+        this.onClick = this.onClick.bind(this);
+    }
+
+    onClick() {
+        if (this.props.submissionData.rules) {
```

Comment:
For some reason the modal is still opening for me. I'm also thinking we could put a disabled property on the `impact-section` div when the total is 0.

<img width="640" alt="Screen Shot 2020-04-03 at 10 06 10 AM" src="https://user-images.githubusercontent.com/7108785/78369566-61d1c080-7593-11ea-80c0-e66c69c9c4af.png">